### PR TITLE
Add functions to jump to middle frames

### PR DESCRIPTION
**Pull request by lucabergamini**
Add 2 functions controlled by `'` and `#` to jump to middle frames.

If obj has annotated frames 2-8-12 pressing:
 - `'` on frame 8 yields 4
 - `#` on frame 8 yields 10

### CODE REVIEW

**Comment by lucabergamini** [comment]
PTAL @stefanopini

File: index.html [hunk 1]
```diff
@@ -1051,6 +1051,43 @@ <h1>vatic.js - A pure Javascript video annotation tool</h1>
               player.seek(nextFrame.frameNumber);
           }
         }
+        else if (e.key === "'") { // [
```
Comment (stefanopini) [inline index.html]: Change key to `,` and update the comment

File: index.html [hunk 2]
```diff
@@ -1051,6 +1051,43 @@ <h1>vatic.js - A pure Javascript video annotation tool</h1>
...
+            }
+            if (prevFrame !== null)
+            {
+              let frame_number = Math.floor((prevFrame.frameNumber + player.currentFrame) / 2);
+              player.seek(frame_number);  // TODO boundaries check?
+            }
+
+          }
+        }
+        else if (e.key === "#") { // [
```
Comment (stefanopini) [inline index.html]: Change key to `.` and update the comment

**Review by stefanopini — CHANGES_REQUESTED** [review]
Thanks!

Please change shortcuts to `,` and `.` to match the arrangement of UK, Italian, and International keyboards and update the comments accordingly.

Please also add a description of the functionality between [line 48 and 49](https://github.com/stefanopini/vatic.js/blob/master/index.html#L48-L49) in the format `<li><span style="color: red; font-weight: bold;">NEW</span> Press "<strong>,</strong>" / "<strong>.</strong>" to ... </li>`

File: index.html [hunk 3]
```diff
@@ -1051,9 +1052,8 @@ <h1>vatic.js - A pure Javascript video annotation tool</h1>
               player.seek(nextFrame.frameNumber);
           }
         }
-        else if (e.key === "'") { // [
+        else if (e.key === ",") { // [
```
Comment (stefanopini) [inline index.html]:
```suggestion
        else if (e.key === ",") { // ,
```

File: index.html [hunk 4]
```diff
@@ -1064,14 +1064,13 @@ <h1>vatic.js - A pure Javascript video annotation tool</h1>
...
             {
               let frame_number = Math.floor((prevFrame.frameNumber + player.currentFrame) / 2);
-              player.seek(frame_number);  // TODO boundaries check?
+              player.seek(frame_number);
             }
 
           }
         }
-        else if (e.key === "#") { // [
+        else if (e.key === ".") { // [
```
Comment (stefanopini) [inline index.html]:
```suggestion
        else if (e.key === ".") { // .
```

**Review by stefanopini — APPROVED** [review]

**Pull request merged**